src/tools/compiletest/src/main.rs

```diff
@@ -809,7 +809,7 @@ fn analyze_gdb(gdb: Option<String>) -> (Option<String>, Option<u32>, bool) {
 
     let gdb_native_rust = version.map_or(false, |v| v >= MIN_GDB_WITH_RUST);
 
-    (Some(gdb.to_owned()), version, gdb_native_rust)
+    (Some(gdb.to_owned()), version, gdb_native_rust && false)
```

Comment:
This looks ... suspicious. Perhaps it was used for local testing?

Reply:
Oops, indeed a mistake!